services/server/server.go

```diff
@@ -203,17 +217,24 @@ func New(ctx context.Context, config *srvconfig.Config) (*Server, error) {
 		required[r] = struct{}{}
 	}
 
-	// Run migration for each configuration version
-	// Run each plugin migration for each version to ensure that migration logic is simple and
-	// focused on upgrading from one version at a time.
-	for v := config.Version; v < srvconfig.CurrentConfigVersion; v++ {
-		for _, p := range plugins {
-			if p.ConfigMigration != nil {
-				if err := p.ConfigMigration(ctx, v, config.Plugins); err != nil {
-					return nil, err
+	if version < srvconfig.CurrentConfigVersion {
+		t1 := time.Now()
+		// Run migration for each configuration version
+		// Run each plugin migration for each version to ensure that migration logic is simple and
+		// focused on upgrading from one version at a time.
+		for v := config.Version; v < srvconfig.CurrentConfigVersion; v++ {
```

Comment:
`v := version`? (`config.Version` seems to be eqaul to  ` srvconfig.CurrentConfigVersion` because of `config.MigrateConfig(ctx)` in the above)

Reply:
Updated, looks like the tests caught this too